apollo-api/src/appleTest/kotlin/com/apollographql/apollo3/ResponseAdapterCacheTest.kt

```diff
@@ -45,7 +45,7 @@ class ResponseAdapterCacheTest {
 }
 
 private fun readInt(cache: ResponseAdapterCache): Int {
-  val adapter = cache.getOperationAdapter("TestQuery") { IntResponseAdapter }
+  val adapter = cache.getAdapterFor(Int::class) { IntResponseAdapter }
```

Comment:
This changed to use a `KClass` key because 2 operations can have the same name if they are in 2 different services